changelogs/fragments/62588-plugins-netconf-ce.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - "plugins/netconf/ce.py: add a dispatch rpc function to send and receive original rpc-reply"
```

I think the following is easier to understand:
```suggestion
  - "Cloudengine netconf plugin - add a dispatch RPC function to send and receive original RPC reply"
```

---

integrated this PR into ansible/ansible#62587